app/src/main/java/com/willowtree/vocable/keyboard/KeyboardFragment.kt

```diff
@@ -140,10 +140,6 @@ class KeyboardFragment : BaseFragment<FragmentKeyboardBinding>() {
 
         (binding.phraseSavedView.root as TextView).setText(R.string.saved_successfully)
 
-        viewModel = ViewModelProviders.of(
```

Comment:
this is probably better bc I think `ViewModelProviders.of` is deprecated now haha